### ISSUE TITLE
Is this truly the model in the paper referenced?

### ISSUE THREAD

**Issue by 24spiders** [Seyed-Ali-Ahmadi/FirePred#2]
The paper says the 400x350 model has "26,915,090 parameters" (Table 7).
This model has 18,207,442 (included in the notebook with the model summary)

What is the cause of this discrepancy?

Thanks!

**Comment by Seyed-Ali-Ahmadi** [comment]
Thanks for mentioning this issue. Yes, the overall structure of the model is the same which is used in the paper. But, during multiple tests and hyperparameter tunings, we had changed some parts and didn't turn them back into the initial stage. 
By the way, here is the correct version which I think you need.

```
def Model_(input_shape1,input_shape2,input_shape3):

    #Get Inputs
    input_1=Input(input_shape1)
    input_2=Input(input_shape2)
    input_3=Input(input_shape3)

    # Hourly features
    out1=hourly_model(input_1)
    # Daily Features
    out2=daily_model(input_2)
    #Constatnt Features
    out3=constant_model(input_3)

    out1=SimpleRNN(64,return_sequences=True,activation='relu') (out1)
    out1=SimpleRNN(32,return_sequences=False,activation='relu') (out1)

    out2=SimpleRNN(64,return_sequences=True,activation='relu') (out2)
    out2=SimpleRNN(32,return_sequences=False,activation='relu') (out2)

    out3=Dense(64,activation='relu') (out1)
    out3=Dense(32,activation='relu') (out1)

    x = concatenate([out1,out2,out3])

    x = Dense(128,activation='relu') (x)
    x = Dense(140000, activation='sigmoid') (x)
    output = tf.keras.layers.Reshape((400,350,1)) (x)
    return Model(inputs=(input_1,input_2,input_3),outputs=output)

inp1=(2,400,350,8)
inp2=(2,400,350,3)
inp3=(400,350,5)

model=Model_(inp1,inp2,inp3)
model.summary()
```

> Total params: 26,915,218 (102.67 MB)
> Trainable params: 26,915,090 (102.67 MB)
> Non-trainable params: 128 (512.00 B)

You can find more details in the PDF file:
[model_corrected.pdf](https://github.com/user-attachments/files/17891805/model_corrected.pdf)


Please let me know if there are further questions.

**Comment by 24spiders** [comment]
I see, and thank you for your reply!

I was wondering about your data process - if each data point is a 'fire day', how did you subsample fire days from the ABoVE dataset? The notebook states:

Test Len: 302
Train Len: 2522

but there are many more than in the ABoVE dataset (which contains thousands of fires which burn for multiple fire days). Was the dataset subsampled? Were large / small fires removed from the dataset prior to rescaling / padding step?

**Comment by 24spiders** [comment]
Thanks!
